site/src/pages/docs/server-annotated-service.mdx

```diff
@@ -1428,3 +1428,71 @@ for more information about RPC and HTTP service names, and how to use it.
 </Tip>
 
 [Publisher]: https://www.reactive-streams.org/reactive-streams-1.0.2-javadoc/org/reactivestreams/Publisher.html
+
+## Kotlin Coroutines Support
+
+You can implement annotated services with Kotlin Coroutines.
```

Comment:
- Let's add a reference link for `Kotlin Coroutines` 
  https://kotlinlang.org/docs/reference/coroutines-overview.html
- Better to describe how to install the Kotlin module?
  ```gradle
  dependencies {
      implementation 'com.linecorp.armeria:armeria-kotlin'
  }
  ```